src/ui/FilterForm.tsx

```diff
@@ -36,6 +37,17 @@ function determineParentFieldValue(subFieldNames, valueByNameForUI) {
 	}, null);
 }
 
+type Props = {
+	error: AnnotationErrorType;
+	feedbackByName: FormFeedbackByName;
+	isSubmitting: boolean;
+	onFieldChange(...args: unknown[]): void;
+	onInitialize(): void;
+	productContent: string;
+	showFeedback: boolean;
+	valueByName: FormValueByName;
```

Comment:
All the values stored in the `valueByName` variable will be boolean, so the type can be changed to `{ [key: string]: boolean }`.